miden/tests/integration/helpers.rs

```diff
@@ -173,3 +192,15 @@ pub fn convert_to_stack(values: &[u64]) -> [Felt; MIN_STACK_DEPTH] {
 pub fn prop_randw<T: proptest::arbitrary::Arbitrary>() -> impl Strategy<Value = Vec<T>> {
     prop::collection::vec(any::<T>(), 4)
 }
+
+pub fn get_proof_options() -> ProofOptions {
+    ProofOptions::new(
+        32,
+        16,
+        0,
+        miden::HashFunction::Blake3_256,
+        miden::FieldExtension::None,
+        8,
+        256,
+    )
+}
```

Comment:
Thanks for pointing this out - I've changed it. I based this on the fibonacci example, which has the options written explicitly. Maybe it would be good to switch that to using default as well?

Reply:
Yep - that would be great!